src/commands/General/Chat Bot Info/help.ts

```diff
@@ -91,12 +103,12 @@ export default class extends SkyraCommand {
 
 	private async _fetchCommands(message: KlasaMessage) {
 		const run = this.client.inhibitors.run.bind(this.client.inhibitors, message);
-		const commands = new Map();
+		const commands = new Collection<string, SkyraCommand[]>();
```

Comment:
There's no reason for this to be `SkyraCommand[]`, not all commands are `SkyraCommand` (built-in ones are normal commands).